scripts/west_commands/tests/test_pyocd.py

```diff
@@ -113,8 +112,7 @@
                                  '-T', str(TEST_TELNET_PORT),
                                  '-t', TEST_TARGET,
                                  '-u', TEST_DEV_ID,
-                                 '-f', TEST_FREQUENCY,
-                                 TEST_TOOL_OPT]
+                                 '-f', TEST_FREQUENCY] + [TEST_TOOL_OPT]
```

Comment:
I'm confused, does this make any difference?

Reply:
No, sorry for cluttering the diff if that's what you're saying. It just looked a little cleaner to my eye and opens up a later patch to this runner to test that multiple arguments are given in order that I wanted to get in after this PR.